### PR TITLE
Get OpProtos in Python

### PR DESCRIPTION
**Pull request by reyoung**
* PyBind and SWIG of paddle cannot be load in a single Python process,
  lazy import all SWIG library of Paddle. Otherwise, the glog, gflags
  are imported twice in a same Python process.
* Note that all PyBind11 return C++ std::string as an unicode. For
  protobuf, it is need be cast to `str` before use them.
* Add unit test for Get `OpProtos`

### CODE REVIEW

File: paddle/pybind/pybind.cc [hunk 1]
```diff
@@ -13,12 +13,18 @@ See the License for the specific language governing permissions and
...
 #include <paddle/framework/scope.h>
 #include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <fstream>
+#include <vector>
 
 namespace py = pybind11;
 namespace pd = paddle::framework;
 
+USE_OP(add_two);
```
Comment (jacquesqiao) [inline paddle/pybind/pybind.cc]: 这个后面是不是会专门放一个地方？

File: python/paddle/v2/optimizer.py [hunk 2]
```diff
@@ -26,6 +24,8 @@ def __impl__():
 
         self.__opt_conf_proto__ = config_parser_utils.parse_optimizer_config(
             __impl__)
+        if swig_api is None:
```
Comment (jacquesqiao) [inline python/paddle/v2/optimizer.py]: 下面这部分修改是另外一个问题？

File: python/paddle/v2/framework/create_op_creation_methods.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import paddle.v2.framework.core as core
+import paddle.v2.framework.proto.op_proto_pb2 as op_proto_pb2
+
+
+def get_all_op_protos():
+    protostrs = core.get_all_op_protos()
+    ret_values = []
+    for pbstr in protostrs:
+        op_proto = op_proto_pb2.OpProto.FromString(str(pbstr))
```
Comment (jacquesqiao) [inline python/paddle/v2/framework/create_op_creation_methods.py]: if this `str()` method the way to fix the Unicode problem?

File: paddle/pybind/pybind.cc (hunk 1 above)
Comment (reyoung) [inline paddle/pybind/pybind.cc]: 是的。可以单独放一个.cc里面，而且那个.cc应该也可以是编译器生成的。

File: python/paddle/v2/optimizer.py (hunk 2 above)
Comment (reyoung) [inline python/paddle/v2/optimizer.py]: It is a bug. Done.

File: python/paddle/v2/framework/create_op_creation_methods.py (hunk 3 above)
Comment (reyoung) [inline python/paddle/v2/framework/create_op_creation_methods.py]: Yep.

**Pull request merged**